packages/experimental-loader/src/stylable-transform-loader.ts

```diff
@@ -85,6 +85,18 @@ const stylableLoader: LoaderDefinition = function (content) {
         stylable.resolver
     );
 
+    try {
```

Comment:
why did you added this here also? not sure about it was there any issue?

Reply:
@dvdovenko ?

Reply:
@barak007 yes, I added it here because typescript types for `this._module` can be undefined / null